File: kotlin/kotlin-vs-java/java-example/src/main/java/org/example/ex07/Exam07.java
package org.example.ex07;

/**
 * data class
 */
public class Exam07 {
    public Exam07() {
        User user = new User();
        user.setName("홍길동");
        user.setEmail("email@email.com");
        user.setAge(20);
    }

    public static void main(String[] args) {
        new Exam07();
    }
}

File: kotlin/kotlin-vs-java/kotlin-example/src/main/kotlin/ex07/Exam07.kt
package ex07

/**
 * data class
 */
fun main() {
    val user = User(name = "홍길동");
    println(user)

    user.age = 20
    user.email = "email@email.com"

    println(user)

    val user2 = user.copy(name = "이순신")
    println(user2)

}